src/components/layout/FSidebar.js

```diff
@@ -113,6 +120,32 @@ And 75% if the viewport is 800px or smaller...
       </f-fade>
   </span>
   `,
+  directives: {
+    'click-outside': {
+      bind: function(el, binding, vNode) {
+        if (typeof binding.value !== 'function') {
+        	const compName = vNode.context.name
+          let warn = `[Vue-click-outside:] provided expression '${binding.expression}' is not a function, but has to be`
+          if (compName) { warn += `Found in component '${compName}'` }
+          
+          console.warn(warn)
+        }
+        const bubble = binding.modifiers.bubble
+        const handler = (e) => {
+          if (bubble || (!el.contains(e.target) && el !== e.target)) {
```

Comment:
if (binding.modifiers.bubble)

Reply:
wouldnt work without `(!el.contains(e.target) && el !== e.target) `
so right now it is 
`if (binding.modifiers.bubble || (!el.contains(e.target) && el !== e.target) )`